web/components/layouts/Main/Main.stories.tsx

```diff
@@ -36,6 +36,10 @@ export default {
   },
 } satisfies ComponentMeta<typeof Main>;
 
+// mock the Websocket to prevent ani webhook calls from being made in storybook
+// @ts-ignore
+window.WebSocket = () => {};
```

Comment:
Does this work? @gabek 
I've not wrapped the component inside, but I've mocked the WebSocket func to not do anything

Reply:
That works!